fixtures/register.json

```diff
@@ -82,13 +88,14 @@
       "cameo/thunder-wash-600-rgb",
       "cameo/thunder-wash-600-w"
     ],
-    "Fan": [
```

Comment:
I didn't change anything but only ran the `generate_register.js` script. However, I guess the categories section is not sorted by category names but by appearance in fixtures (which are sorted alphabetically, though). "Strobe" is also before "Blinder".